src/main/java/org/opendatakit/aggregate/client/popups/ConfirmExportDeletePopup.java

```diff
@@ -16,81 +16,70 @@
 
 package org.opendatakit.aggregate.client.popups;
 
+import static org.opendatakit.aggregate.client.security.SecurityUtils.secureRequest;
+import static org.opendatakit.aggregate.constants.common.ExportStatus.ABANDONED;
+import static org.opendatakit.aggregate.constants.common.ExportStatus.AVAILABLE;
+
 import com.google.gwt.safehtml.shared.SafeHtmlBuilder;
+import com.google.gwt.user.client.ui.FlexTable;
+import com.google.gwt.user.client.ui.HTML;
 import org.opendatakit.aggregate.client.AggregateUI;
 import org.opendatakit.aggregate.client.SecureGWT;
 import org.opendatakit.aggregate.client.form.ExportSummary;
 import org.opendatakit.aggregate.client.widgets.AggregateButton;
 import org.opendatakit.aggregate.client.widgets.ClosePopupButton;
-import org.opendatakit.aggregate.constants.common.ExportStatus;
-
-import com.google.gwt.event.dom.client.ClickEvent;
-import com.google.gwt.event.dom.client.ClickHandler;
-import com.google.gwt.user.client.rpc.AsyncCallback;
-import com.google.gwt.user.client.ui.FlexTable;
-import com.google.gwt.user.client.ui.HTML;
 
 /**
  * Popup asking for confirmation to delete an exported data file
- * 
+ *
  * @author wbrunette@gmail.com
  * @author mitchellsundt@gmail.com
- * 
  */
 public final class ConfirmExportDeletePopup extends AbstractPopupBase {
 
-  private static final String BUTTON_ICON = "<img src=\"images/green_right_arrow.png\" />";
+  private static final String BUTTON_ICON = "<img src=\"images/green_right_arrow.png\"/>";
   private static final String TOOLTIP_TXT = "Remove this exported datafile";
-  private static final String HELP_BALLOON_TXT = "This will remove this exported datafile.  You will no longer "
-      + "be able to download this datafile.";
-
-  private final ExportSummary export;
```

Comment:
Removing these fields is a nice side effect.

Reply:
Yup! Clean up season! :)